react-charts/bar-chart/step5.md

```diff
@@ -61,8 +66,6 @@ It should look like this:
 &lt;ChartAxis dependentAxis showGrid label=&quot;Percentage&quot;/&gt;
 </pre>
 
-- This will add grid lines which appear behind the ChartBar

```

Comment:
Ensuring grid lines appear behind the bars is important to the design. They can get mixed up if ChartAxis is added below ChartBar.

Reply:
Ensuring grid lines appear behind the bars is important to the design. They can get mixed up if ChartAxis is added below ChartBar.

Perhaps we can add a different message in a follow up PR?